src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.MaxClassName.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Windows.Win32
+{
+    internal static partial class PInvoke
+    {
+        /// <summary>
+        /// <para>The maximum length for lpszClassName is 256. If lpszClassName is greater than the maximum length, the RegisterClassEx function will fail.</para>
+        /// <para><see href="https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassexw#members">Read more on docs.microsoft.com</see>.</para>
```

Comment:
```suggestion
        /// <para><see href="https://learn.microsoft.com/windows/win32/api/winuser/ns-winuser-wndclassexw#members">Read more on docs.microsoft.com</see>.</para>
```

https://review.learn.microsoft.com/help/platform/links-how-to?branch=main#locale-codes-in-external-links